tests/api/test_picking.py

```diff
@@ -10,23 +10,23 @@
 import pytest
 from fgpyo.sequence import reverse_complement
 
+from prymer.api import FilteringParams
+from prymer.api import MinOptMax
+from prymer.api import Primer
+from prymer.api import PrimerPair
+from prymer.api import Span
+from prymer.api import Strand
+from prymer.api import build_and_pick_primer_pairs
+from prymer.api import build_primer_pairs
+from prymer.api import pick_top_primer_pairs
 from prymer.api.melting import calculate_long_seq_tm
-from prymer.api.minoptmax import MinOptMax
-from prymer.api.picking import FilteringParams
 from prymer.api.picking import _dist_penalty
 from prymer.api.picking import _seq_penalty
-from prymer.api.picking import build_and_pick_primer_pairs
-from prymer.api.picking import build_primer_pairs
 from prymer.api.picking import check_primer_overlap
 from prymer.api.picking import is_acceptable_primer_pair
-from prymer.api.picking import pick_top_primer_pairs
 from prymer.api.picking import score as picking_score
```

Comment:
Note that the remaining functions are not included in the `api` namespace, and must be imported from the appropriate submodule. 

IMO these functions should either be added to the `api` namespace, or marked as private with a leading underscore, but I don't think that needs to fall within the scope of this PR